README.md

```diff
@@ -16,12 +16,16 @@ Navigate to the [**CONFIGURE** tab](https://docs.viam.com/build/configure/) of y
 
 ## Configure your `viamrtsp` camera
 
-Copy and paste the following attributes template into the JSON configuration:
+1. Add the `viamrtsp:onvif` discovery service.
+1. Use the test tab to find available cameras.
+1. Copy (one of) the returned configurations.
+1. Add the `viamrtsp` camera.
+1. Paste the copied configuration attributes into the JSON configuration:
 
 ```json
 {
    "rtp_passthrough": true,
-   "rtsp_address": "rtsp://foo:bar@192.168.10.10:554/stream"
+   "rtsp_address": "rtsp://tavy16d.viam.local/stream"
```

Comment:
To the degree anyone will type these addresses in by hand, I think* keeping port 554 would be better -- that's the default port for rtsp connections:
```
"rtsp_address": "rtsp://tavy16d.viam.local:554/stream"
```

It might be that the underlying libraries we use try it by default, but I couldn't say with certainty.